src/theme/src/default-theme/component-specific/getTooltipProps.js

```diff
@@ -1,11 +1,21 @@
 import tinycolor from 'tinycolor2'
 import palette from '../foundational-styles/palette'
 
-const getTooltipProps = () => {
-  return {
-    backgroundColor: tinycolor(palette.neutral.base)
-      .setAlpha(0.95)
-      .toString()
+const getTooltipProps = appearance => {
+  switch (appearance) {
+    case 'card':
```

Comment:
This might be a misunderstanding of the semantics, but I've often followed the rule of thumb that a `Tooltip` is simple, short content. It cannot contain html or other components, only strings. With those semantics, things that look like a raised "card" would qualify as a `Popover` IMO.

Not sure if this meshes with Evergreen's component semantics, but thought it was worth mentioning here (if it allows us to simplify things).

Reply:
The distinction currently between `Tooltip` and `Popover` is the interaction kind: hover vs. click. We might want to consider supporting something like `<Popover interaction={InteractionKind.HOVER} />` similar to BlueprintJS at one point. I think we should leave that for a future major release.